content/docs/next-release/operator.md

```diff
@@ -0,0 +1,666 @@
+```
+
+After the role is granted, switch back to a non-privileged user.
+
+Jaeger Agent can be configured to be deployed as a `DaemonSet` using a `HostPort` to allow Jaeger clients in the same node to discover the agent. In OpenShift, a `HostPort` can only be set when a special security context is set. A separate service account can be used by the Jaeger Agent with the permission to bind to `HostPort`, as follows:
+
+```bash
+oc create -f https://raw.githubusercontent.com/jaegertracing/jaeger-operator/master/deploy/examples/openshift/hostport-scc-daemonset.yaml # <1>
+oc new-project myappnamespace
+oc create -f https://raw.githubusercontent.com/jaegertracing/jaeger-operator/master/deploy/examples/openshift/service_account_jaeger-agent-daemonset.yaml # <2>
+oc adm policy add-scc-to-user daemonset-with-hostport -z jaeger-agent-daemonset # <3>
+oc apply -f https://raw.githubusercontent.com/jaegertracing/jaeger-operator/master/deploy/examples/openshift/agent-as-daemonset.yaml # <4>
+```
+<1> The `SecurityContextConstraints` with the `allowHostPorts` policy
+<2> The `ServiceAccount` to be used by the Jaeger Agent
+<3> Adds the security policy to the service account
+<4> Creates the Jaeger Instance using the `serviceAccount` created in the steps above
+
+{{< warning >}}
+Without such a policy, errors like the following will prevent a `DaemonSet` to be created: `Warning FailedCreate 4s (x14 over 45s) daemonset-controller Error creating: pods "agent-as-daemonset-agent-daemonset-" is forbidden: unable to validate against any security context constraint: [spec.containers[0].securityContext.containers[0].hostPort: Invalid value: 5775: Host ports are not allowed to be used`
+{{< /warning >}}
+
+After a few seconds, the `DaemonSet` should be up and running:
+
+```bash
+$ oc get daemonset agent-as-daemonset-agent-daemonset
+NAME                                 DESIRED   CURRENT   READY     UP-TO-DATE   AVAILABLE
+agent-as-daemonset-agent-daemonset   1         1         1         1            1
+```
+
+# Quick Start - Deploying the AllInOne image
+
+The simplest possible way to create a Jaeger instance is by creating a YAML file like the following example.  This will install the default AllInOne strategy, which deploys the "all-in-one" image (agent, collector, query, ingestor, Jaeger UI) in a single pod, using in-memory storage by default.
+
+{{< info >}}
+This default strategy is intended for development, testing, and demo purposes, not for production.
+{{< /info >}}
+
+```yaml
+apiVersion: jaegertracing.io/v1
+kind: Jaeger
+metadata:
+  name: simplest
+```
+
+The YAML file can then be used with `kubectl`:
+<!-- TODO - Add OKD commands and tabs shortcode. -->
+
+```bash
+kubectl apply -f simplest.yaml
+```
+
+In a few seconds, a new in-memory all-in-one instance of Jaeger will be available, suitable for quick demos and development purposes. To check the instances that were created, list the `jaeger` objects:
+
+```bash
+$ kubectl get jaeger
```

Comment:
I think it's common to issue the `get` commands using the object in plural: `kubectl get jaegers`